docker/webui/run_openqa.sh

```diff
@@ -7,9 +7,36 @@ else
   /usr/share/openqa/script/upgradedb --user geekotest --upgrade_database
 fi
 
+function scheduler(){
+  su -u /usr/share/openqa/script/openqa-scheduler-daemon
```

Comment:
`su -u` does not look sensible to me. Should be either `su geekotest` or `sudo -u geekotest`

Reply:
Yes, that looks wrong. I'd also prefer using sudo. All services should definitely run as the same user.